src/status_im/ui/screens/wallet/onboarding/setup/views.cljs

```diff
@@ -0,0 +1,49 @@
+(ns status-im.ui.screens.wallet.onboarding.setup.views
+  (:require-macros [status-im.utils.views :as views])
+  (:require [clojure.string :as string]
+            [re-frame.core :as re-frame]
+            [status-im.i18n :as i18n]
+            [status-im.react-native.resources :as resources]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.styles :as components.styles]
+            [status-im.ui.screens.wallet.components :as comp]
+            [status-im.ui.screens.wallet.onboarding.setup.styles :as styles]
+            [status-im.ui.components.bottom-buttons.view :as bottom-buttons]
+            [status-im.ui.components.button.view :as button]
+            [status-im.utils.utils :as utils]))
+
+(defn signing-word [word]
+  [react/view styles/signing-word
+   [react/text {:style           styles/signing-word-text
+                :font            :roboto-mono
```

Comment:
Maybe we should consider iOS?

Reply:
`RobotoMono-Regular` should also used on iOS (according to the existing designs)